src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorCustomRuleObject.cs

```diff
@@ -76,12 +76,6 @@ public class NewAzureRmFrontDoorCustomRuleObject : AzureFrontDoorCmdletBase
         [Parameter(Mandatory = false, HelpMessage = "Rate limit thresold")]
         public int? RateLimitThreshold { get; set; }
 
-        /// <summary>
```

Comment:
Note: please also add a change description to the changelog - this is the trigger for updating the version and releasing the module